tools/check-help.sh

```diff
@@ -39,3 +39,9 @@ if ! ("$BINARY" --no-such-parameter 2>&1 1>/dev/null || :) | grep . >/dev/null;
     echo "$(basename "$BINARY") with an unknown parameter does not print to stderr"
     exit 4
 fi
+
+# --help and -h are equivalent
+if ! cmp <("$BINARY" -h) <("$BINARY" --help); then
```

Comment:
Maybe use `diff` instead of `cmp`, so we see what's different? That should be especially helpful in CIs.